### ISSUE TITLE
Let's make a game!

### ISSUE THREAD

**Issue by erlend-sh** [erlend-sh/good-south#1]
This is going to be a "living document". It'll have more structure added to it over time; for now I'm gonna start with a quick braindump.

---

I believe there are many eventually-obvious things we can do as a game-building venture to increase our chances of success (and we'll get back to how we can define _success_ together).

Merely making a *good game* does not guarantee any kind of commercial success. Let alone a commercial success great enough to fund *more games* and *more indie game developers*.

# Goals

1. Make an MVP ready for sale.

More ambitious goals will soon follow.

# We are building a community

We are building a modding game. Another word for that is "community game". This game lives and dies by its community. So building the community = building the game. I expand on the business side of this in [Open Source Games Thesis](https://docs.google.com/document/d/1_jUR63Fjhr-WZ3IJ9DFYVyLq09iMB5rBnGWyI41NCqo/edit#).

We are a Godot game. Being able to tap into that community is incredibly valuable and must be utilized to its fullest potential.

I will have loooots more to say about this later.

## Discord

https://www.youtube.com/watch?v=Zg7tRh0k_a8

## DevBlogging

Every time we talk in public about our game, it increases our potential for success. Every public posting is another potential customer and/or contributor. I'd like us to work together to come up with a weekly devlog that we can make together.

**Examples:**
- http://blog.wolfire.com/
- https://factorio.com/blog/
- https://veloren.net/ (pure Rust game)
- https://ratwizard.dev/dev-log/antorum (Rust + Unity game by a Minecraft developer)
- *Suggest yours*

# Openness is our strength

For a moddable game, the more open it is the more appealing it is to its prospective community of modders. Modders want *access*. The more the better.

Public sharing can be anything from a source-available ["not for commercial use" license](https://polyformproject.org/licenses/noncommercial/1.0.0/), to a more traditional and permissive open source license like Apache 2 or [BOL](https://blueoakcouncil.org/2019/03/06/model.html) for code that doesn't need any sort of protection.

Continued topics:
- An open client?
- Open source utilities
- Default licenses for mods
- Openness as community building

# Ego is the enemy

I stole that title from [a book of the same name](https://www.amazon.com/Ego-Enemy-Ryan-Holiday/dp/1591847818). The book only scratches the surface of the understanding of ego as a mental model and the benefits of (momentary) ego-death, but it's a good starter. I think there's a pretty good summary of the book here:

https://medium.com/@ameet/why-ego-is-the-enemy-and-what-we-can-do-about-it-a4eae45a81d8

In short: If we ever get into some kind of disagreement or misunderstanding that gets so bad that it feels genuinely painful, we got our egos mixed up with *the work* somehow. The more we can agree on at this moment for our shared reason to be here while the work is easy, the more we'll be able to remember our present selves when the work gets so hard that we forget  why we we started doing it in the first place. ❤️
